### DISCUSSION TITLE
what is the difference between filter and interpolator?

### DISCUSSION THREAD

**Discussion by vetyalmao** [OpenTabletDriver/OpenTabletDriver#1310] [Q&A] [answered]
look at the name

**Comment by X9VoiD** [comment, marked as answer]
Filters and interpolators are basically the same, the difference is that interpolators have the ability to process reports asynchronously, which makes it possible to do "fake report rates". This is pretty much what Hawku/Devocub drivers do to "change" tablet report rate.
